api/config/config.env

```diff
@@ -0,0 +1,6 @@
+PORT=4000
+DB_NAME=postgres
+DB_USER=postgres
+DB_PASSWORD=postgres
+DB_PORT=5432
+DB_HOST=localhost
```

Comment:
Settings of "config.env" isn't required in GitHub repository.
They can be different in other developers.

Reply:
I pushed it so that every member has example of conffig. After all of them change it to their variables it will be added to git ignore file.